docs/node/run-your-node/sentry-node.md

```diff
@@ -56,86 +56,49 @@ Before using this configuration you should collect the following information to
   on the validator node. Note that the above command is only available in `oasis-node` from version 20.8.1 onward.
 
 ```yaml
-##
-# Oasis Sentry Node Configuration
-#
-# This file's configurations are derived from the command line args found in the
-# root command of the oasis-node binary. For more information execute:
-#
-#     oasis-node --help
-#
-# Settings on the command line that are separated by a dot all belong to the
-# same nested object. So "--foo.bar.baz hello" would translate to:
-#
-#     foo:
-#       bar:
-#         baz: hello
-##
-
-# Set this to where you wish to store node data. The node artifacts
-# should also be located in this directory (for us this is /node/data)
-datadir: /node/data
-
-# Logging.
-#
-# Per-module log levels are defined below. If you prefer just one unified log
-# level, you can use:
-#
-# log:
-#   level: debug
-log:
-  level:
-    # Per-module log levels. Longest prefix match will be taken. Fallback to
-    # "default", if no match.
-    default: debug
-    tendermint: warn
-    tendermint/context: error
-  format: JSON
-  # By default logs are output to stdout. If you're running this in docker keep
-  # the default
-  #file: /var/log/oasis-node.log
-
-# Path to the genesis file for the current version of the network.
+mode: client
+common:
+    data_dir: /node/data
+    log:
+        format: JSON
+        level:
+            cometbft: warn
+            cometbft/context: error
+            # Per-module log levels. Longest prefix match will be taken. Fallback to
+            # "default", if no match.
+            default: debug
+            # By default logs are output to stdout. If you're running this in docker 
+            # keep the default
+            #file: /var/log/oasis-node.log
+consensus:
+    external_address: tcp://{{ external_address }}:26656
+    listen_address: tcp://0.0.0.0:26656
+    prune:
```

Comment:
I would remove pruning here as well since it's not really necessary and we usually don't have it configured in these example configs.